scripts/tvm_cli/README.md

```diff
@@ -0,0 +1,48 @@
+# TVM Command Line Interface
+
+A command line tool that compiles neural network models using
+[TVM](https://github.com/apache/incubator-tvm).
+
+## Usage
+
+Build a docker image that contains all the dependencies and scripts needed to
+run the tool.
+
+```bash
+$ # From root of the model zoo repo
+$ docker build -f scripts/tvm_cli/Dockerfile \
+               -t autoware-model-zoo/tvm_cli:local \
+               scripts/tvm_cli
+```
+
+The CLI can now be invoked as a container
+
+```bash
+$ docker run -it --rm -v `pwd`:`pwd` -w `pwd` \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    autoware-model-zoo/tvm_cli:local -h
+```
```

Comment:
When running this as a test, I get the following output at the top:

```
2020-10-08 15:18:00.785374: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-10-08 15:18:00.785405: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
```

I added `--gpus all` since I have both an Nvidia graphics card and the `nvidia-container-toolkit` installed. It seems like this would also require CUDA to be installed in the container to support Nvidia GPUs (and maybe ROCm for AMD GPUs). Do we want to add these to the container?

Reply:
I'm currently working on a subsequent patch, part of this patch is adding CUDA support to the TVM CLI. In my opinion we should leave the current patch as-is (w.r.t CUDA) because we're adding CUDA on a future patch anyway.
Please let me know what are your thoughts about this.